src/components/homepage/carousel.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <el-carousel trigger="click" :interval="5000" arrow="none" id="carousel">
+    <el-carousel-item v-for="item in 4" :key="item"></el-carousel-item>
+  </el-carousel>
+</template>
+
+<script>
+export default {
+  name: "Carousel"
+};
+</script>
+
+<style>
+#carousel {
+  border-radius: 4px;
+}
+
+.el-carousel__item:nth-child(2n) {
+  background-color: #99a9bf;
+}
+
+.el-carousel__item:nth-child(2n + 1) {
+  background-color: #d3dce6;
+}
+</style>
```

Comment:
You didn't add any thing here yet, did you?

Reply:
Yeah, maybe later you should add a picture function in the back-end
leave it here, maybe change it after

Reply:
ok